documentation/deployments.md

```diff
@@ -11,3 +11,57 @@ Upon merging a PR to `main` a deployment to the development environment will be
 1. Initialise Terraform
 1. Run the Terraform plan
 1. Apply the Terraform changes
+
+### PaaS
+
+Some useful things to know about PaaS...
+
+GOV.UK Platform as a Service (PaaS) is a cloud-hosting platform built by the Government Digital Service (GDS). GOV.UK PaaS manages the deployment of your apps, services and background tasks so you don’t need to hire people with specialist cloud skills.
+
+GOV.UK PaaS uses the open source Cloud Foundry project, and runs on Amazon Web Services.
+
+This repo has an organisation setup on PaaS. Speak to the org manager to authorise the [creation of a personal account](https://docs.cloud.service.gov.uk/get_started.html#get-started)
+
+The Cloud Coundry CLI can be used to manage apps hosted on PaaS - [Setup Cloud Foundry CLI](https://docs.cloud.service.gov.uk/get_started.html#set-up-the-cloud-foundry-command-line)
+
+### Environments
+
+This application runs through four environments:
+
+- dev
+- test
+- pre prod
+- prod
+
+`dev` is deployed to automatically when a PR is merged into the `main` branch.
+
+`test`, `pre prod` and `prod` deployments are triggered manually from the Github workflow tab
+
+Each environment has a dedicated PaaS user account to run CI/CD jobs using Github workflows.
+
+To find out the user account details, contact the org manager.
+
+### Seed data
+
+[Seed data is destroyed](../db/seeds.rb) and recreated in `dev`, `test` and `pre prod` environments.
+
+`prod` does not get seeded with data automatically.
```

Comment:
so we may still need to populate production somehow with a script? like we were discussing adding users earlier

Reply:
Don't think this has been decided yet and it's out of scope for this PR